src/devices/fakeDepthCamera/fakeDepthCameraDriver.cpp

```diff
@@ -40,6 +40,8 @@ fakeDepthCameraDriver::~fakeDepthCameraDriver()
 bool fakeDepthCameraDriver::open(Searchable& config)
 {
     Property cfg;
+    cfg.fromString(config.toString());
```

Comment:
Thanks for this! Had to hard-code things here for debugging a couple of days ago (bug turned out to be on my side).